hack/verify-gendocs.sh

```diff
@@ -104,8 +73,10 @@ set -e
 rm -rf "${TMP_DOCROOT}"
 if [ $ret -eq 0 ]
 then
-	echo "${DOCROOT} up to date."
+  echo "${DOCROOT} up to date."
 else
-	echo "${DOCROOT} is out of date. Please run hack/run-gendocs.sh"
-	exit 1
+  echo "${DOCROOT} is out of date. Please run hack/run-gendocs.sh"
```

Comment:
If there is an outdated file that should be removed, the diff against `.files_generated` will fail, but run-gendocs.sh will not fix that. Maybe diff `.files_generated` separately and suggest deleting outdated files if generated doesn't match?